composer.json

```diff
@@ -9,6 +9,7 @@
         "composer/composer": "^1.0",
         "doctrine/collections": "~1.2",
         "gitonomy/gitlib": "~1.0",
+        "phptype/array": "^1",
```

Comment:
The trade-off is loss of clarity and single-responsibility. However, if it is preferred, I can make the change you suggest and substitute that code directly in lieu of this dependency.